nextjs-nodejs-mongodb/api/guestbook/list.js

```diff
@@ -0,0 +1,42 @@
+// Import dependencies
+const connect = require("../../lib/db");
+
+module.exports = async (req, res) => {
+  // Set caching headers to serve stale content (if over a second old)
+  // while revalidating fresh content in the background. 
```

Comment:
```suggestion
  // while revalidating fresh content in the background
```

Most of your comments do not end with a full stop.